docs/CHANGELOG.md

```diff
@@ -50,6 +51,16 @@
 
 --- 
 
+## 7.3.4
+
+#### What's New?
+* Syncing with latest codebase - currently up to sync with 8.0.1.
+
+#### Changes
+* See [8.0.1](#801) changes
```

Comment:
8.0.1 here?

Reply:
Yeah, I'm basically keeping 7.x in sync with master which was last released for 8.0.1. I thought we're not publishing any ansible/docker changes in 7.x that is more up-to-date than 8.x?

Reply:
Makes sense.  Thanks